src/njoy21/input/DTFR/Card3/Ipingp/test/Ipingp.test.cpp

```diff
@@ -0,0 +1,35 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "DTFR ipingp values",
+  "[DTFR],[Card3], [Ipingp]"){
+  GIVEN( "a valid iptotl value" ){
+    Argument< DTFR::Card3::Iptotl >iptotl; iptotl.value = 6;
```

Comment:
I prefer to have these two commands on separate lines. @apmccartney prefers then on the same line. The Style Guide doesn't explicitly state how to do this, so you can pick for yourself.